3421.xml

```diff
@@ -0,0 +1,618 @@
+			</Item>
+			<Item ID="8">
+				<Name>Low current</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Set to True when current is below low current threshold.</Description>
+			</Item>
+			<Item ID="9">
+				<Name>High current threshold</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>V</Units>
+				<Description>Threshold above which the high current resource is set to True.</Description>
+			</Item>
+			<Item ID="10">
+				<Name>High current</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Set to True when current is above high current threshold.</Description>
+			</Item>
+			<Item ID="11">
+				<Name>Frequency</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>Hz</Units>
+				<Description>Instantaneous frequency measured.</Description>
+			</Item>
+			<Item ID="12">
+				<Name>Angle of I-U</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>deg</Units>
+				<Description>Instantaneous phase angle measured.</Description>
+			</Item>
+			<Item ID="13">
+				<Name>Instantaneous Power Factor</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Instantaneous power factor overall.</Description>
+			</Item>
+			<Item ID="14">
+				<Name>Low power factor threshold</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Threshold below which the low power factor resource is set to True.</Description>
+			</Item>
+			<Item ID="15">
+				<Name>Low power factor</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Set to True when power factor is below low power factor threshold.</Description>
+			</Item>
+			<Item ID="16">
+				<Name>Active power+ (QI+QIV)</Name>
```

Comment:
Is the `(QI+QIV)` part important in the name of the resource? The special characters may cause problems in systems that will use the name as identifier in databases etc. Also might be confusing in UIs.

This is perhaps something for IPSO WG to consider for guidelines on object / resource names.